libass/ass.h

```diff
@@ -24,7 +24,7 @@
 #include <stdarg.h>
 #include "ass_types.h"
 
-#define LIBASS_VERSION 0x01702000
+#define LIBASS_VERSION 0x01702010
```

Comment:
This bump is exactly as it should be; but unless we want to include them in the hotfix 0.17.3 merging this now will again bring up the question of how to name the release.

`LIBASS_VERSION` has the lowest nibble reserved for hotfixes, so it’s technically fine — but we have no scheme to map this to public release numbers and since usually `LIBASS_VERSION` corresponds to the public release number, naming `0x01702001` 0.17.3 will be a bit confusing

Reply:
I'm not in any rush with this; happy to delay if needed. Alternately, if we want to just include it in the hotfix (seems fairly low-risk?), we can increment to `0x01703000` at that time.

Reply:
so current non-cosmetic, non-CI changes from 0.17.2 are:
 - upgrading x86inc to be more generic about calls (replacing checkasm-specific Haiku fix)
 - explicit AC_CANONICAL_HOST
 -  use memchr for pasing tags
 - fontconfig refcounting and ordering

And to be added at least: fix shared-only checkasm

Which seems fine for tagging things on `master` again rather than a release branch. In that case this and `-fno-errno-math` are prob fine too for emrge ahead of 0.17.2